server/game/cards/07-WotW/IkomaAnakazu.js

```diff
@@ -0,0 +1,35 @@
+const DrawCard = require('../../drawcard.js');
+const EventRegistrar = require('../../eventregistrar.js');
+const AbilityDsl = require('../../abilitydsl');
+const { EventNames } = require('../../Constants');
+
+class IkomaAnakazu extends DrawCard {
+    setupCardAbilities() {
+        this.brokenProvincesThisPhase = {};
+        this.eventRegistrar = new EventRegistrar(this.game, this);
+        this.eventRegistrar.register([EventNames.OnBreakProvince, EventNames.OnPhaseEnded]);
+
+        this.persistentEffect({
+            condition: context => context.source.isParticipating() && this.brokenProvincesThisPhase[context.source.controller.opponent.name] > 0,
```

Comment:
This will throw an error in single player games.  Also, you probably don't need to track the number of provinces, just having a boolean is fine. I guess it doesn't really matter though